src/main/java/org/splitec/client/DatabaseClient.java

```diff
@@ -0,0 +1,85 @@
+package org.splitec.client;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import  org.splitec.config.CosmosConfig;
+import java.util.ArrayList;
+
+public class DatabaseClient {
+
+    private CosmosClient client;
+
+    private final String databaseName = "CalamaDB";
+    private final String containerName = "Users";
+
+    private CosmosDatabase database;
+    private CosmosContainer container;
+
+    public void close() {
+        client.close();
+    }
+
+    public String getDatabaseName() {
+        return databaseName;
+    }
+
+
+    public CosmosContainer getContainer() {
+        return container;
+    }
+
+    public void buildClient() {
+        try {
+            System.out.println("Using Azure Cosmos DB endpoint: " + CosmosConfig.HOST);
```

Comment:
@luucasfl ?????????????????????????????????????????????????????????????????????? why are you loggin that in the console? please implement a logger class

Reply:
Done